src/helpers/helpers.canvas.js

```diff
@@ -174,9 +174,9 @@ var exports = {
 
 	/**
 	 * Returns true if the point is inside the rectangle
-	 * @param {Object} point - The point to test
-	 * @param {Object} area - The rectangle
-	 * @returns {Boolean}
+	 * @param {object} point - The point to test
```

Comment:
` Point ` is used as `{x, y}` in other places

Reply:
Not sure what you mean? (we don't define `Point` in our doc)

Reply:
If so, `{Point}` in [this line](https://github.com/chartjs/Chart.js/blob/8ef67a6a516c4299af409efc5fd7eae7b45c4c20/src/core/core.interaction.js#L9) (and other places) should be replaced with `{object}`.

Reply:
We also use it as a type in the `interactions` file. I may be wrong and `Point` may be defined somewhere?!. If not, I would replace all uses by `{object}` and define `IPoint` later during a deeper jsdoc cleaning / reorganization.

Reply:
I've changed the others to `{object}`